samples/Mvc.LocalizationSample.Web/Resources/Models.Product.fr.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="MaxLengthTwo" xml:space="preserve">
+    <value>{0} doivent avoir une longueur d'au plus deux.</value>
```

Comment:
This is definitly not the patern we want to encourage, but it's also the format the other model resources are using. I filed #176 to resolve for all instances where it's happened and keep this PR relativly on subject.